README.md

```diff
@@ -42,63 +42,70 @@ After you have all the prerequisites installed and have configured your
     EOF
     ```
 
-4. Initialize Terraform (only needed once)
+4. If necessary, set the Makefile variables `DO_INSTANCE_TAGNAME` and
```

Comment:
Nope. I broke it down so that that it would be easier to keep multiple versions of experiment.mk, for different experiments, without having to touch Makefile, which should remain untouched.